src/Components/TimeSheets/Form/index.js

```diff
@@ -179,78 +181,73 @@ function Form() {
           </div>
           <div className={styles.box}>
             <label>Task</label>
-            <select
+            <Select
               name="task"
+              placeholder="Select a task"
               required
-              value={inputTimeSheetValue.task}
-              onChange={onChangeInputValue}
-            >
-              <option value="" disabled hidden>
-                Select a task
-              </option>
-              {tasks.map((task) => {
-                return (
-                  <option placeholder="taskhere" key={task._id} value={task._id}>
-                    {task.description}
-                  </option>
-                );
-              })}
-            </select>
+              onSelect={onChangeInputValue}
+              data={tasks.map((task) => ({
+                id: task._id,
+                value: task.description
+              }))}
+              value={
+                inputTimeSheetValue.task !== ''
+                  ? tasks.find((task) => task._id === inputTimeSheetValue.task)._id
+                  : undefined
+              }
```

Comment:
To avoid use undefined I suggest
```suggestion
              value={
                inputTimeSheetValue.task !== '' && (
                   tasks.find((task) => task._id === inputTimeSheetValue.task)._id
                  )
              }
```